neo/Prompt/Commands/Send.py

```diff
@@ -13,16 +13,77 @@
+                                       fee=framework[2], owners=framework[3], user_tx_attributes=framework[4])
+        return framework
+
+    def command_desc(self):
+        p1 = ParameterDesc('assetId or name', 'the asset you wish to send')
+        p2 = ParameterDesc('address', 'the NEO address you will send to')
+        p3 = ParameterDesc('amount', 'the amount of the asset you wish to send')
+        p4 = ParameterDesc('--from-addr={addr}', 'specify the NEO address you wish to send from', optional=True)
+        p5 = ParameterDesc('--fee={priority_fee}', 'attach a fee to give your tx priority (> 0.001)', optional=True)
+        p6 = ParameterDesc('--owners=[{addr}, ...]', 'specify tx owners', optional=True)
+        p7 = ParameterDesc('--tx-attr=[{"usage": <value>,"data":"<remark>"}, ...]', 'specify unique tx attributes', optional=True)
+        params = [p1, p2, p3, p4, p5, p6, p7]
+        return CommandDesc('send', 'send an asset', params=params)
+
+
+class CommandWalletSendMany(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        framework = construct_send_many(PromptData.Wallet, arguments)
+        if type(framework) is list:
+            return process_transaction(PromptData.Wallet, contract_tx=framework[0], scripthash_from=framework[1], scripthash_change=framework[2],
+                                       fee=framework[3], owners=framework[4], user_tx_attributes=framework[5])
+        return framework
+
+    def command_desc(self):
+        p1 = ParameterDesc('number of outgoing tx', 'the number of tx you wish to send')
+        p2 = ParameterDesc('--change-addr={addr}', 'specify the change address', optional=True)
+        p3 = ParameterDesc('--from-addr={addr}', 'specify the NEO address you wish to send from', optional=True)
+        p4 = ParameterDesc('--fee={priority_fee}', 'attach a fee to give your tx priority (> 0.001)', optional=True)
+        p5 = ParameterDesc('--owners=[{addr}, ...]', 'specify tx owners', optional=True)
+        p6 = ParameterDesc('--tx-attr=[{"usage": <value>,"data":"<remark>"}, ...]', 'specify unique tx attributes', optional=True)
+        params = [p1, p2, p3, p4, p5, p6]
+        return CommandDesc('sendmany', 'send multiple contract transactions', params=params)
+
+
+class CommandWalletSign(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        jsn = get_arg(arguments)
+        return parse_and_sign(PromptData.Wallet, jsn)
+
+    def command_desc(self):
+        p1 = ParameterDesc('jsn', 'transaction in JSON format')
+        params = [p1]
+        return CommandDesc('sign', 'sign multi-sig tx', params=params)
+
+
 def construct_send_basic(wallet, arguments):
-    if not wallet:
-        print("please open a wallet")
-        return False
     if len(arguments) < 3:
         print("Not enough arguments")
-        return False
+        return
```

Comment:
Because `construct_send_basic` can return `[contract_tx, scripthash_from, fee, owners, user_tx_attributes]` it should return `None` or `False` in early returns.

Reply:
@LysanderGG 
@ixje has explained to me that it is undesirable to have a function return both a boolean and say a function. Instead if you just leave return blank, it returns none, which will also be interpreted as False, so all my tests pass.

Reply:
I agree for `boolean` and something else (I suggested `False` because it was this way before).
I think it should return `None` explicitly then.

When I see `return` (nothing) I assume that the function doesn't return anything. It can be misleading imo.

@ixje any thoughts?

Reply:
I'm indifferent when it comes to `return` or `return None`. I guess I'm used to the idea that `return` always returns `None` under the hood, so to speak. Might as well make it explicit if that clarifies it for all future readers. I'll probably need to slapping on the fingers to remind me of doing that though ;)

Reply:
The add a few more words regarding return values; at some point I expect to start applying gradual typing. We've had past cases where functions were returning `None`, `bool` or a `tuple`. That's just not workable.

Reply:
Thanks for the answer, I won't make this as a blocker for this PR, so @jseagrave21 as you want :smile:

Reply:
I would like to leave it this way so the format is consistent with #716. Consistency is what matters most to me so people will be able to follow our example throughout the code.